docs/en/security/encrypt.md

```diff
@@ -1,51 +1,149 @@
+
+- Before writing to an object storage, data blocks are compressed using LZ4 or ZStandard.
 - A random 256-bit symmetric key `S` and a random seed `N` are generated for each data block.
-- AES-GCM-based encryption of each data block using `S` and `N` yields `encrypted_data`.
+- Each data block is encrypted using `S` and `N` based on AES-GCM encryption yielding `encrypted_data`.
 - To avoid the symmetric key `S` from being transmitted in clear text over the network, the symmetric key `S` is encrypted with the RSA key `M` to obtain the ciphertext `K`.
 - The encrypted data `encrypted_data`, the ciphertext `K`, and the random seed `N` are combined into an object and then written to the object storage.
 
 The steps for decrypting the data are as follows:
+
 - Read the entire encrypted object (it may be a bit larger than 4MB).
 - Parse the object data to get the ciphertext `K`, the random seed `N`, and the encrypted data `encrypted_data`.
 - Decrypt `K` with RSA key to get symmetric key `S`.
 - Decrypt the data `encrypted_data` based on AES-GCM using `S` and `N` to get the data block plaintext.
 - Decompress the data block.
 
 
-### Key Management
+### Enable Data Encryption At Rest
+
+:::note
+Data Encryption At Rest must be enabled when creating file system. The file system that has areadly enabled Data Encryption At Rest does not need to enable it again. 
+:::
+
+There are following steps to enable Data Encryption At Rest:
+
+1. Create a RSA private key
+2. Run the encrypted file system that is created by the RSA private key
+3. Mount the file system
+
+#### Step 1: create a RSA private key
+
+The RSA private key plays a crtical role in Data Encryption At Rest, usually manually generated by OpenSSL. The following command generates a 2048-bit RSA private key with a file name of `my-priv-key.pem` in the currect directory using aes256 algorithm.
+
+```shell
+openssl genrsa -out my-priv-key.pem -aes256 2048
+```
+Because of the aes256 algorithm, a passphrase (at least 4 bits) is required for this RSA private key. The passphrase can be considered as a password for encrypting the file of the RSA private key, which is also the last layer of security assurance for the RSA private key.
+
+:::caution
+The security of RSA private key is critical when Data Encryption At Rest encryption is enabled. Once the key is compromised, the data can confront security risk. If the key is lost, **all the encrypted data will be lost and cannot be recovered**!
+:::
+
+#### Step 2: create an encrypted file system
+
+The option `--encrypt-rsa-key` is required to specify RSA private key when creating an encrypted file system. The provided private key content will be written to the metadata engine. Since passphrase is mandatory in the ase256-encrypted RSA private key, the environmental variable `JFS_RSA_PASSPHRASE` is required to specify the passphrase of the private key before creaging and mounting file system.
```

Comment:
```suggestion
The option `--encrypt-rsa-key` is required to specify RSA private key when creating an encrypted file system. The provided private key content will be written to the metadata engine. Since passphrase is mandatory in the ase256-encrypted RSA private key, the environment variable `JFS_RSA_PASSPHRASE` is required to specify the passphrase of the private key before creating and mounting file system.
```